tests/unittests/test_client.py

```diff
@@ -6,20 +6,22 @@
 from datetime import datetime
 import calendar
 
+@mock.patch("tap_linkedin_ads.client.LinkedinClient.write_access_token_to_config")
 @mock.patch("requests.Session.post")
 class TestLinkedInClient(unittest.TestCase):
 
-    def test_access_token_empty_expires(self, mocked_post):
+    def test_access_token_empty_expires(self, mocked_post, mock_write_token):
         '''
         Ensure that we retrieve and set expires for client with no self.__expires
         '''
-        client = _client.LinkedinClient('client_id', 'client_secret', 'refresh_token', 'access_token')
+        client = _client.LinkedinClient('client_id', 'client_secret', 'refresh_token', 'access_token', 'config_path')
 
-        future_time = int(datetime.utcnow().timestamp()) + 100
+        future_time = int(datetime.utcnow().timestamp()) + 88400
```

Comment:
if we change the timedelta in fetch_and_set per the above comment, this duration can be reduced accordingly